docs/uds-package-practices.md

```diff
@@ -82,6 +82,29 @@ To help maintain a UDS Package, it:
 
 - **Must** release its package to the `ghcr.io/defenseunicorns/packages/<group>` namespace as the application's name (i.e. `ghcr.io/defenseunicorns/packages/uds/mattermost`)
 
+## Versioning
+You can use this section to decide how best to version a UDS Package.
```

Comment:
```suggestion
Use this section to decide how best to version a UDS Package.
```

Not a strong suggestion but I love the active voice and not giving room to make new choices as we engineer.  I love the idea of evolving this document but this section should always define how we version UDS Packages.